clio-client/src/test/resources/testdata/metadata

```diff
@@ -1,33 +1,33 @@
 {
-    "analysisType" : "analysisType",
-    "baitIntervals" : "baitIntervals",
-    "dataType" : "dataType",
-    "individualAlias" : "individualAlias",
-    "initiative" : "initiative",
-    "lcSet" : "lcSet",
-    "libraryType" : "libraryType",
-    "machineName" : "machineName",
-    "molecularBarcodeName" : "mbName",
-    "molecularBarcodeSequence" : "mbSequence",
-    "pairedRun" : true,
-    "productFamily" : "productFamily",
-    "productName" : "productName",
-    "productOrderId" : "productOrderId",
-    "productPartNumber" : "productPartNumber",
-    "project" : "project",
-    "readStructure" : "readStructure",
-    "researchProjectId" : "researchProjectId",
-    "researchProjectName" : "researchProjectName",
-    "rootSampleId" : "rootSampleId",
-    "runDate" : "2017-08-09T14:21:57.995-04:00",
-    "runName" : "runName",
-    "sampleAlias" : "sampleAlias",
-    "sampleGender" : "sampleGender",
-    "sampleId" : "sampleId",
-    "sampleLsid" : "sampleLsid",
-    "sampleType" : "sampleType",
-    "targetIntervals" : "targetIntervals",
-    "ubamMd5" : "ubamMd5",
-    "ubamPath" : "ubamPath",
-    "ubamSize" : 1
+  "analysis_type" : "analysisType",
+  "bait_intervals" : "analysisType",
```

Comment:
Before the values were different for each field, now they're all the same. Why was this changed?

Reply:
I think this had to do with how the json was being unmarshalled. I was getting leftover keys on any value that had camel case instead of snake case